pages/page.py

```diff
@@ -98,15 +98,29 @@ def ajax_complete(self, driver):
 
         return not jquery_active
 
-    def click(self, element, scroll=True):
+    def click(self, element, scroll=True, click_attempts_max=2):
         """
         Click on the target element. If scroll is True, this will attempt to
-        scroll the element into view then click on it
-        """
-        if scroll:
-            self.scroll_to_element(element)
+         scroll the element into view then click on it.
 
-        element.click()
+        If the click attempt fails with WebDriverException then retry
+         scroll/click until the total number of attempts equals the max number
+         of attempts
+        """
+        attempts = 0
+        while attempts < click_attempts_max:
+            attempts += 1
```

Comment:
This seems to me more like a C code than python. What about using:

`for attempts in range(click_attempts_max):` 

This way, you won't have to explicitly zero or increment the "attempts" variable.